letsencrypt/cli.py

```diff
@@ -1040,8 +1040,7 @@ def main(cli_args=sys.argv[1:]):
 
     # note: arg parser internally handles --help (and exits afterwards)
     plugins = plugins_disco.PluginsRegistry.find_all()
-    parser, tweaked_cli_args = create_parser(plugins, cli_args)
-    args = parser.parse_args(tweaked_cli_args)
```

Comment:
I'm not sure if I understand. `tweaked_cli_args` wasn't used anywhere other than the call to `parser.parse_args(tweaked_cli_args)` and the value of `args` that `configuration.NamespaceConfig(args)` uses should be identical to the previous version (if it's not, I introduced a bug). Am I not understanding something?